src/components/AddressSearch.js

```diff
@@ -68,19 +75,27 @@ const defaultProps = {
     value: undefined,
     defaultValue: undefined,
     containerStyles: [],
+    isLimitedToUSA: true,
     renamedInputKeys: {
         street: 'addressStreet',
         city: 'addressCity',
         state: 'addressState',
         zipCode: 'addressZipCode',
     },
+
+    // A maximum of -1 means the user can enter as many characters as they want
+    maxInputLength: -1,
```

Comment:
bugzero: This default doesn't work well in native and caused [this regression](https://github.com/Expensify/App/issues/14918). We replaced this value by `undefined`.